doc/reference/orm.rst

```diff
@@ -723,50 +818,15 @@ Method decorators
 
 .. todo:: With sphinx 2.0 : autodecorator
 
-.. _reference/api/onchange:
-
-onchange: updating UI on the fly
---------------------------------
-
-When a user changes a field's value in a form (but hasn't saved the form yet),
-it can be useful to automatically update other fields based on that value
-e.g. updating a final total when the tax is changed or a new invoice line is
-added.
-
-* computed fields are automatically checked and recomputed, they do not need
-  an ``onchange``
-* for non-computed fields, the :func:`~odoo.api.onchange` decorator is used
-  to provide new field values::
-
-    @api.onchange('field1', 'field2') # if these fields are changed, call method
-    def check_change(self):
-        if self.field1 < self.field2:
-            self.field3 = True
-
-  the changes performed during the method are then sent to the client program
-  and become visible to the user
-
-* Both computed fields and onchanges are automatically called by the
-  client without having to add them in views
-* It is possible to suppress the trigger from a specific field by adding
-  ``on_change="0"`` in a view::
-
-    <field name="name" on_change="0"/>
+.. todo:: Add in Views reference
```

Comment:
ok fine, but you know my general feeling
![do_do_not](https://user-images.githubusercontent.com/9530767/65887588-7a6fdc80-e39e-11e9-9802-daf303b7273b.jpg)